packages/react/src/MinimalFooter/MinimalFooter.tsx

```diff
@@ -274,21 +274,22 @@ function SocialLogomarks({socialLinks, logoHref}: SocialLogomarksProps) {
   )
 }
 
-type LinkProps = {
-  href: string
-} & BaseProps<HTMLAnchorElement>
+type LinkProps<C extends React.ElementType> = BaseProps<C> & {as?: C} & Omit<React.ComponentPropsWithoutRef<C>, keyof C>
```

Comment:
Could you update the prop table in docs too please? 

<img width="1013" alt="Screenshot 2023-05-13 at 16 58 44" src="https://github.com/primer/brand/assets/13340707/11516d9a-e8b4-4e4f-876a-6ed4d883c577">